projects/projects.yml

```
@@ -52,7 +52,7 @@ mathematica:
 
 topos:
   description: |
-    This is a WIP project to define some topos theory within the Lean theorem prover.
+    This repository contains formal verification of results in Topos Theory, drawing from "Sheaves in Geometry and Logic" and "Sketches of an Elephant". It currently includes a range of constructions and theorems about both Grothendieck topoi and elementary toposes.
```

```suggestion
    This repository contains formal verifications of results in Topos Theory, drawing from "Sheaves in Geometry and Logic" and "Sketches of an Elephant". It currently includes a range of constructions and theorems about both Grothendieck topoi and elementary toposes.
```
or "a formal verification"? Or "This repository formally verifies results..."?

Is there a distinction between the plural forms "topoi" and "toposes"?

---

For the first, either works for me! For the second, that was meant as a little joke, I'm happy to remove it

---

Guess the joke went over my head, but that's fine!